testsuite/MDAnalysisTests/core/test_atomselections.py

```diff
@@ -74,6 +74,13 @@ def test_protein(self, universe):
         assert_equal(sorted(sel.indices),
                      sorted(universe.s4AKE.atoms.indices),
                      "selected protein is not the same as auto-generated protein segment s4AKE")
+    
+    def test_resnames(self, universe):
+        for x in MDAnalysis.core.selection.ProteinSelection.prot_res:
+            universe.atoms.residues[0].resname = x
```

Comment:
Is `universe` a function-level fixture or module level?

If you change it then it has to become function-level, otherwise you're changing the universe for other methods. In this case I'd suggest to create a second fixture, e.g., `universe2`, and make that function level (or maybe @tylerjereddy @kain88-de @richardjgowers have a better idea)?

Reply:
yes, this fixture is class level. We shouldn't change it here. Having another universe that doesn't have the `scope='class'` argument would be good for that.

Reply:
So is it sufficient to use u2 = MDAnalysis.Merge(universe.atoms) and work with that? How can I specify the Universe's scope?

Reply:
@sobuelow you can just use `universe_copy` instead of `universe`

Reply:
`assert myprot.atoms.ix == sel.ix` should then be `assert_equal(myprot.atoms.ix, sel.ix)` because you're comparing arrays, aren't you?